service/src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -652,57 +634,11 @@ private WsmResource constructResource(DbResource dbResource) {
     }
   }
 
-  @Transactional(
-      propagation = Propagation.REQUIRED,
-      isolation = Isolation.SERIALIZABLE,
-      readOnly = true)
   private DbResource getDbResource(String sql, MapSqlParameterSource params) {
```

Comment:
Should this one be `@ReadTransaction` too?

Reply:
nope - this is a private method called by `getResource` and `getResourceByName`, both of which are annotated with `@ReadTransaction`.